plugins/action/k8s_info.py

```diff
@@ -359,7 +359,12 @@ def run(self, tmp=None, task_vars=None):
         # find the file in the expected search path
         src = self._task.args.get("src", None)
 
-        if src:
+        if (
+            src
+            and not src.startswith("http://")
```

Comment:
```suggestion
            and not src.startswith(("http://", "https://", "ftp://")):
```
`startswith` takes a tuple as well. Not a blocker, just a hint for the future.